lib/new_relic/agent/instrumentation/aws_sdk_firehose/instrumentation.rb

```diff
@@ -0,0 +1,66 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module Firehose
+    INSTRUMENTED_METHODS = %w[
+      create_delivery_stream
+      delete_delivery_stream
+      describe_delivery_stream
+      list_delivery_streams
+      list_tags_for_delivery_stream
+      put_record
+      put_record_batch
```

Comment:
Is it worth asking if these should be message broker segments? They seem very similar to the ones we switched in Kinesis.

Reply:
Definitely worth asking! I will do this

Reply:
Per spec, Firehose has no message broker segments. So we're good here!